### PR TITLE
Make API and Open source more obvious to users [OSF-6710]

### PR DESCRIPTION
**Pull request by Johnetordoff**
## Purpose

To make the API more obvious to users, add link to API docs in footer and a link the API docs in help/support.

## Changes

changes the static html in the page footer reflect suggestions.

## Ticket

https://openscience.atlassian.net/browse/OSF-6710

### CODE REVIEW

File: website/templates/footer.mako [hunk 1]
```diff
@@ -6,9 +6,9 @@
                 <ul>
                     <li><a href="${domain}explore/activity/">Explore</a></li>
                     <li><script type="text/javascript">document.write("<n uers=\"znvygb:pbagnpg@bfs.vb\" ery=\"absbyybj\">Pbagnpg</n>".replace(/[a-zA-Z]/g,function(e){return String.fromCharCode((e<="Z"?90:122)>=(e=e.charCodeAt(0)+13)?e:e-26)}));</script><noscript>Contact OSF: <span class="obfuscated-email-noscript"><strong><u>cont<span style="display:none;">null</span>act@<span style="display:none;">null</span>osf.<span style="display:none;">null</span>io</u></strong></span></noscript></li>
-                    <li><a href="${domain}support">Help/Support</a></li>
-                    <li><a href="http://help.osf.io" target="_blank" rel="noreferrer">Guides</li>
-                    <li><a href="${domain}faq">FAQ</a></li>
+                    <li><a href="${domain}support">Guides/FAQ</a></li>
+                    <li><a href="https://api.osf.io/v2/docs/">API</a></li>
+                    <li><a href="https://github.com/CenterForOpenScience/osf.io">Souce Code</a></li>
```
Comment (sloria) [inline website/templates/footer.mako]: Typo here. "Source"

File: website/templates/public/pages/support.mako [hunk 2]
```diff
@@ -80,6 +80,7 @@
                 <a href="https://groups.google.com/forum/#!forum/openscienceframework" class="btn btn-link"><i class="fa fa-users"></i> Join our mailing list </a>
                 <a href="https://www.facebook.com/OpenScienceFramework" class="btn btn-link"><i class="fa fa-facebook"></i> Follow us on Facebook </a>
                 <a href="https://github.com/centerforopenscience" class="btn btn-link"><i class="fa fa-github"></i> Connect with COS on Github</a>
+                <a href="https://api.osf.io/v2/docs/" class="btn btn-link"><i class="fa fa-cogs"></i> Access our powerful dev facing API.</a>
```
Comment (sloria) [inline website/templates/public/pages/support.mako]: I don't see this in the ticket. Was this requested by Sara or Nici?

Comment (Johnetordoff) [inline website/templates/public/pages/support.mako]: "we should include a link to API docs in footer **AND** a link the API docs in help/support", I missed at first too.

Comment (Johnetordoff) [inline website/templates/public/pages/support.mako]: @sloria This was requested by @niipfeiffer is it g2g now?

Comment (sloria) [inline website/templates/public/pages/support.mako]: I see. Thanks for the clarification.

Let's simplify the language. Also, remove the period, for consistency.

> Access our developer API

File: website/templates/footer.mako [hunk 3]
```diff
@@ -6,9 +6,9 @@
                 <ul>
                     <li><a href="${domain}explore/activity/">Explore</a></li>
                     <li><script type="text/javascript">document.write("<n uers=\"znvygb:pbagnpg@bfs.vb\" ery=\"absbyybj\">Pbagnpg</n>".replace(/[a-zA-Z]/g,function(e){return String.fromCharCode((e<="Z"?90:122)>=(e=e.charCodeAt(0)+13)?e:e-26)}));</script><noscript>Contact OSF: <span class="obfuscated-email-noscript"><strong><u>cont<span style="display:none;">null</span>act@<span style="display:none;">null</span>osf.<span style="display:none;">null</span>io</u></strong></span></noscript></li>
-                    <li><a href="${domain}support">Help/Support</a></li>
-                    <li><a href="http://help.osf.io" target="_blank" rel="noreferrer">Guides</li>
-                    <li><a href="${domain}faq">FAQ</a></li>
+                    <li><a href="${domain}support">Guides/FAQ</a></li>
```
Comment (brianjgeiger) [inline website/templates/footer.mako]: This does not match the ticket. Please double check and make it exactly what was requested.

File: website/templates/public/pages/support.mako [hunk 4]
```diff
@@ -80,6 +80,7 @@
                 <a href="https://groups.google.com/forum/#!forum/openscienceframework" class="btn btn-link"><i class="fa fa-users"></i> Join our mailing list </a>
                 <a href="https://www.facebook.com/OpenScienceFramework" class="btn btn-link"><i class="fa fa-facebook"></i> Follow us on Facebook </a>
                 <a href="https://github.com/centerforopenscience" class="btn btn-link"><i class="fa fa-github"></i> Connect with COS on Github</a>
+                <a href="https://api.osf.io/v2/docs/" class="btn btn-link"><i class="fa fa-cogs"></i> Access our powerful dev facing API</a>
```
Comment (brianjgeiger) [inline website/templates/public/pages/support.mako]: I talked with @saradbowman and this was added by mistake. Please remove this change; it's being handled in help.osf.io.

**Comment by brianjgeiger** [comment]
Two minor changes and this should be good. Pass complete. 🐧

**Pull request merged**